tools/uninstall.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+# vim: tabstop=4 softtabstop=4 shiftwidth=4 noexpandtab
+###############################################################################
+# This is the Linux installation script for the King Phisher Client and
+# Server on supported distributions.
+#
+# Project Home Page: https://github.com/securestate/king-phisher/
+# Authors:
+#   Erik Daguerre
+#
+# Supported Linux Distributions:
+#   Linux Flavor  | Client | Server |
+#   --------------|--------|--------|
+#   BackBox		  |   yes  |   yes  |
+#   CentOS		  |   no   |   yes  |
+#   Debian		  |   yes  |   yes  |
+#   Fedora		  |   yes  |   yes  |
+#   Kali		  |   yes  |   yes  |
+#   Ubuntu		  |   yes  |   yes  |
+#
+###############################################################################
+
+E_USAGE=64
+E_SOFTWARE=70
+E_NOTROOT=87
+FILE_NAME="$(dirname $(readlink -e $0) 2>/dev/null)/$(basename $0)"
+GIT_CLONE_URL="https://github.com/securestate/king-phisher.git"
+if [ -z "$KING_PHISHER_DIR" ]; then
+	KING_PHISHER_DIR="/opt/king-phisher"
+fi
+LINUX_VERSION=""
+
+answer_all_no=false
+answer_all_yes=false
+
+function prompt_yes_or_no {
+	# prompt the user to answer a yes or no question, defaulting to yes if no
+	# response is entered
+	local __prompt_text=$1
+	local __result_var=$2
+	if [ "$answer_all_no" == "true" ]; then
+		$__result_var="no";
+		return 0;
+	elif [ "$answer_all_yes" == "true" ]; then
+		eval $__result_var="yes";
+		return 0;
+	fi
+	while true; do
+		read -p "$__prompt_text [Y/n] " _response
+		case $_response in
+			"" ) eval $__result_var="yes"; break;;
+			[Yy]* ) eval $__result_var="yes"; break;;
+			[Nn]* ) eval $__result_var="no";  break;;
+			* ) echo "Please answer yes or no.";;
+		esac
+	done
+	return 0;
+}
+
+function show_help {
+	echo "Usage: $(basename $0) [-h] [-n/-y]"
+	echo ""
+	echo "King Phisher Install Script"
+	echo ""
+	echo "optional arguments"
+	echo "  -h, --help			show this help message and exit"
```

Comment:
These options should all be spaces instead of hard tabs. That way it will always be consistent, that's also how the install.sh script has them defined, it looks like these got switched to hard tabs.